docs/topics/mta-web-config-maven-repo.adoc

```diff
@@ -8,6 +8,11 @@
 
 You can configure a Maven repository in the *Repositories* view of the {ProductName} ({ProductShortName}) {WebName}.
 
+[NOTE]
+====
+This procedure is possible only if the `rwx_supported` configuration option of the Tackle CR is set to `true`. Otherwise, the *Consume insecure artifact repositories* switch is disabled.
```

Comment:
Need to double check this, if I remember correctly - this is not affected by `rwx_supported` parameter value

Reply:
@ibragins 
The screenshot that you sent me shows that the switch is disabled.